packages/angular_devkit/build_optimizer/src/transforms/wrap-enums.ts

```diff
@@ -322,7 +322,13 @@ function findStatements(
 
   for (let index = statementIndex + 1; index < statements.length; ++index) {
     const statement = statements[index];
+
     if (!ts.isExpressionStatement(statement)) {
+      if (ts.isEmptyStatement(statement)) {
```

Comment:
Can you add a comment here explaining that this is a workaround for ngcc's current handling of class expressions and is not needed to support TS directly?

Reply:
Done